app/models/concerns/audit_extensions.rb

```diff
@@ -31,6 +31,16 @@ def ensure_taxonomies_not_escalated
     end
   end
 
+  def is_auditable_taxable?
+    auditable.class.included_modules.include?(Taxonomix) ||
+    (auditable.respond_to?(:organization) || auditable.respond_to?(:location)) ||
+    (audited_changes && (!audited_changes.include?('location_id') || !audited_changes.include?('organization_id')))
+  end
+
+  def set_current_taxonomy
+    super if is_auditable_taxable?
```

Comment:
in fact I think we don't want to set taxonomies based on current scope regardless of the resource being taxable, this method might be handy in the second fix but this is just about ignoring current scope of newly created audits

when I'm thinking about the migration now, I think we can't reconstruct the data correctly, we don't have to provide that either (again, keep the code somewhere, might become handy in the second fix)